Jenkinsfile

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+node(label: 'ubuntu') {
+    catchError {
+        def environmentDockerImage
+
+        def dockerTag = env.BUILD_TAG.replace('%2F', '-')
```

Comment:
why's this needed?  The tags have the form `jenkins-brooklyn-library-master-docker-179`

Reply:
@geomacy this is needed because if the job name contains a `/` (which is possible as the name will be the actual branch to build) it will be replaced by the URL escaped version, i.e. `%2F`. However, docker tag accepts only alphanumerical characters + `.`, `-` and `_`

Reply:
btw `%2F` is `/`, but that's what you intend here, I guess? Just checking.